fuzzbunny.js

```diff
@@ -198,14 +209,14 @@ function highlightsFromRanges(targetStr, ranges) {
 }
 
 /**
- * fuzzyMatchSanitized is called by fuzzyMatch, it's a slightly lower level call
+ * fuzzyScoreItem is called by fuzzyMatch, it's a slightly lower level call
  * If perf is of importance and you want to avoid lowercase + trim + highlighting on every item
  * Use this and only call highlightsFromRanges for only the items that are displayed
  * @param {string} targetStr - lowercased trimmed target string to search on
  * @param {string} searchStr - lowercased trimmed search string
  * @returns {{score: number, ranges: number[]} | null} - null if no match
  */
-function fuzzyMatchSanitized(targetStr, searchStr) {
+function fuzzyScoreItem(targetStr, searchStr) {
```

Comment:
Note: this is a breaking change in the interface. I don't expect many users of this function but renamed it to more accurately reflect what it does.